.github/workflows/windows-build.yaml

```diff
@@ -7,7 +7,7 @@ on:
 
   # Run every day at 12pm UTC
   schedule:
-    - cron: "0 12 * * *" # The main branch cadence. This runs every day at 12pm UTC.
+    - cron: "0 12 * * 1" # This runs once a week at 12pm UTC.
```

Comment:
I'd prefer once a day so we knew it broke earlier, but can we ensure it's like on Tuesday or Wednesday then?

Reply:
Sounds good, changing it to every Wednesday for now 😄